datafusion/expr/src/window_frame.rs

```diff
@@ -90,10 +90,7 @@ impl TryFrom<ast::WindowFrame> for WindowFrame {
                         | WindowFrameBound::Following(Some(v))
                             if *v > 0 =>
                         {
-                            Err(DataFusionError::NotImplemented(format!(
-                                "With WindowFrameUnits={}, the bound cannot be {} PRECEDING or FOLLOWING at the moment",
-                                units, v
-                            )))
+                            Ok::<(), DataFusionError>(())
                         }
                         _ => Ok(()),
```

Comment:
This code now always returns OK, right? it seems like the check on `bound` can be removed entirely, though perhaps I  am missing something. 


```rust
        } else {
            let units = value.units.into();
            Ok(Self {
                units,
                start_bound,
                end_bound,
            })
        }
```